### PR TITLE
[READY FOR REVIEW] Ongoing blinkr fixes

### PR DESCRIPTION
**Pull request by LightGuard**

### CODE REVIEW

**Comment by paulrobinson** [comment]
Can you prefix the title with '[HOLD]' if it's not ready for review. Otherwise let me know and I'll take a look.

**Comment by LightGuard** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

File: _config/blinkr.yaml [hunk 1]
```diff
@@ -28,9 +31,10 @@ skips:
 #    # JDF-770
   - !ruby/regexp /^https:\/\/docs.jboss.org\/author\/display\/PLINK\/SAML\+Authenticators\+\%28Tomcat\%2CJBossAS\%29+/
 ignores:
-  - url: !ruby/regexp /boms/
-  - url: !ruby/regexp /archetypes/
-  - url: !ruby/regexp /archetypes/
+  #- url: !ruby/regexp /boms/
+  #- url: !ruby/regexp /archetypes/
+  #- url: !ruby/regexp /archetypes/
+  - url: !ruby/regexp /get-started/
```
Comment (LightGuard) [inline _config/blinkr.yaml]: Needs to come out

File: _config/blinkr.yaml [hunk 2]
```diff
@@ -1,12 +1,15 @@
 skips:
   - !ruby/regexp /developer-materials/
-  - !ruby/regexp /bom/
-  - !ruby/regexp /archetype/
+  - !ruby/regexp /get-started/
```
Comment (LightGuard) [inline _config/blinkr.yaml]: Needs to be removed

**Comment by LightGuard** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

File: _config/site.yml [hunk 3]
```diff
@@ -93,7 +93,8 @@ profiles:
     dcp_base_url: http://dcpbeta-searchisko.rhcloud.com/
     dcp2_base_url: http://dcp2-searchisko.rhcloud.com/
     dcp2_base_protocol_relative_url: //dcp2-searchisko.rhcloud.com/
-    metrics: true
+    metrics: false
+    adobe_metrics_url: //assets.adobedtm.com/2b327a0a54b320bf1bcdb5fa39a2b896027067d9/satelliteLib-9ec3c34e9d660fa2fcb85ced512c3a48dc09c0c5-staging.js
```
Comment (paulrobinson) [inline _config/site.yml]: Why is this config required? The plan was to have 'metrics' default to 'false'.

Comment (paulrobinson) [inline _config/site.yml]: Ignore the above, not thinking things through.

**Comment by LightGuard** [comment]
retest this please

File: _config/blinkr.yaml [hunk 4]
```diff
@@ -108,7 +111,12 @@ ignores:
   - url: https://community.jboss.org/emailPasswordToken%21input.jspa #Not an error
   - url: https://developer.jboss.org/people/nzhu #Not an error
   - url: https://developer.jboss.org/people/mmusgrov #Not an error
+  - url: http://devops.com/2015/06/03/redhat-culture-keeping-devops-faith/ #Not an error
 # ---------------BELOW ARE CURRENT OUTSTANDING ISSUES--------------------------
+  - url: http://www.redhat.com/rhecm/rest-rhecm/jcr/repository/collaboration/jcr:system/jcr:versionStorage/e9457db80a05260166324e2c81027566/1/jcr:frozenNode/rh:pdfFile.pdf #RHD-718
```
Comment (paulrobinson) [inline _config/blinkr.yaml]: This is now fixed.

File: _config/site.yml (hunk 3 above)
Comment (paulrobinson) [inline _config/site.yml]: You don't need this if metrics is set to false.

File: _config/site.yml [hunk 5]
```diff
@@ -109,7 +110,8 @@ profiles:
     dcp2_base_protocol_relative_url: //dcp.stage.jboss.org/
     keycloak_account_url: https://developers.stage.redhat.com/auth/realms/rhd/account/
     keycloak_auth_url: https://developers.stage.redhat.com/auth/
-    metrics: true
+    metrics: false
+    adobe_metrics_url: //assets.adobedtm.com/2b327a0a54b320bf1bcdb5fa39a2b896027067d9/satelliteLib-9ec3c34e9d660fa2fcb85ced512c3a48dc09c0c5-staging.js
```
Comment (paulrobinson) [inline _config/site.yml]: You don't need this if metrics is set to false.

File: _config/site.yml [hunk 6]
```diff
@@ -126,7 +128,8 @@ profiles:
     dcp2_base_protocol_relative_url: //dcp.stage.jboss.org/
     keycloak_account_url: https://it-developers.stage.redhat.com/auth/realms/rhd/account/
     keycloak_auth_url: https://it-developers.stage.redhat.com/auth/
-    metrics: true
+    metrics: false
+    adobe_metrics_url: //assets.adobedtm.com/2b327a0a54b320bf1bcdb5fa39a2b896027067d9/satelliteLib-9ec3c34e9d660fa2fcb85ced512c3a48dc09c0c5-staging.js
```
Comment (paulrobinson) [inline _config/site.yml]: You don't need this if metrics is set to false.

File: _eap-quickstarts [hunk 7]
```diff
@@ -1 +1 @@
-Subproject commit 9c2b50a2b68a9d1f9f4fba22d0799964990add9c
+Subproject commit 71a4719e5f52f25857b51b3a4f2e781db5659f45
```
Comment (paulrobinson) [inline _eap-quickstarts]: This change is now in master

File: _config/site.yml [hunk 8]
```diff
@@ -142,6 +145,7 @@ profiles:
     dcp2_base_url: http://dcp2.jboss.org/
     require_tag: true
     metrics: true
+    adobe_metrics_url: //assets.adobedtm.com/2b327a0a54b320bf1bcdb5fa39a2b896027067d9/satelliteLib-9ec3c34e9d660fa2fcb85ced512c3a48dc09c0c5.js
```
Comment (paulrobinson) [inline _config/site.yml]: How does this relate to dtm_mode?

File: _ext/jboss_developer.rb [hunk 9]
```diff
@@ -110,28 +110,27 @@ def transform site, page, content
...
+                        a['href'] = File.join page.metadata[:browse], url
+                        ## One off for petclinic
+                        #if url == 'CHANGES.md' && page.output_path.include?('spring-petclinic')
+                        #  a['href'] = File.join page.metadata[:browse], url
+                        #end
+
+                        ## One off for unified push issues
+                        #if url == 'CHANGES.md' && page.output_path.include?('spring-petclinic')
+                        #  a['href'] = File.join page.metadata[:browse], url
+                        #end
```
Comment (paulrobinson) [inline _ext/jboss_developer.rb]: Should remove the commented out code if we are not likely to add it back soon.

File: _layouts/base.html.slim [hunk 10]
```diff
@@ -251,6 +234,22 @@ body(class="#{page.url.gsub('/','').split('.').first || 'home'} #{page.body_clas
...
+    javascript:
+      dataLayer = [{'channel' : 'developer'}];
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+              new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+              j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+              '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-NJWS5L');
+    noscript
+      iframe src="//www.googletagmanager.com/ns.html?id=GTM-NJWS5L" height="0" width="0"
+
```
Comment (paulrobinson) [inline _layouts/base.html.slim]: Adobe insist that we put this straight after the body. Unless we have a good reason not to, we should do what they ask.

File: _config/site.yml [hunk 11]
```diff
@@ -139,7 +139,7 @@ profiles:
     dcp2_base_protocol_relative_url: //dcp.stage.jboss.org/
     keycloak_account_url: https://it-developers.stage.redhat.com/auth/realms/rhd/account/
     keycloak_auth_url: https://it-developers.stage.redhat.com/auth/
-    metrics: true
+    metrics: false
```
Comment (paulrobinson) [inline _config/site.yml]: We need metrics on for this build. Notice that it's using the staging version of the dtm_code

File: _portal-quickstarts [hunk 12]
```diff
@@ -1 +1 @@
-Subproject commit caf178764214cd14a6c93b65308343945f755cd1
+Subproject commit 8f36f2774cfd5d878042bb13bfd592ac87e3e9f7
```
Comment (paulrobinson) [inline _portal-quickstarts]: Update to c69f464

File: _sandbox-quickstarts [hunk 13]
```diff
@@ -1 +1 @@
-Subproject commit b5714e49dd1bb5e60233506b62f98b2ea5b88c0a
+Subproject commit 482fcf9ef4a2da8f043ba5984e549c591ab90340
```
Comment (paulrobinson) [inline _sandbox-quickstarts]: Update to 482fcf9ef4a2da8f043ba5984e549c591ab90340

File: _unifiedpush-quickstarts [hunk 14]
```diff
@@ -1 +1 @@
-Subproject commit 65a94db58c07e48cb3fcda4861dba1f119673ec8
+Subproject commit 40324d4a51077124e840df4ca51698cc3bcd70a6
```
Comment (paulrobinson) [inline _unifiedpush-quickstarts]: Correct.

File: _wfk-quickstarts [hunk 15]
```diff
@@ -1 +1 @@
-Subproject commit 3b63ce3a7a3dfddd9b1456a73f6a5c8b2c83e9de
+Subproject commit f3f3742d123aefcfaa369e4fc9d5e9dc37a8d0d4
```
Comment (paulrobinson) [inline _wfk-quickstarts]: Correct.

**Pull request merged**